customers-service/src/main/java/com/petclinic/customers/businesslayer/PhotoServiceImpl.java

```diff
@@ -0,0 +1,97 @@
+package com.petclinic.customers.businesslayer;
+
+import com.petclinic.customers.customerExceptions.exceptions.NotFoundException;
+import com.petclinic.customers.datalayer.*;
+import com.petclinic.customers.presentationlayer.PetRequest;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class PhotoServiceImpl implements PhotoService{
+    private final PhotoRepository photoRepository;
+    private final OwnerRepository ownerRepository;
+    private final PetRepository petRepository;
+
+    public PhotoServiceImpl(PhotoRepository photoRepository, OwnerRepository ownerRepository, PetRepository petRepository) {
+        this.petRepository = petRepository;
+        this.ownerRepository = ownerRepository ;
+        this.photoRepository = photoRepository;
+    }
+
+    @Override
+    public String setPhotoOwner(Photo photo, int ownerId) {
```

Comment:
Did you consider making this method named `replaceOwnerPhoto` because it might better describe what the method is doing